scripts/netContain/Dockerfile

```diff
@@ -13,19 +13,18 @@
 #limitations under the License.
 ##
 
-# One Container for OVS / netplugin / netmaster 
+# One Container for OVS / netplugin / netmaster
 
-FROM ubuntu:16.04
+FROM alpine:3.6
```

Comment:
Please update associated issue with this requirement, and reference that issue in this PR.

Reply:
I'm not sure what issue I should point to here.